### DISCUSSION TITLE
Интеграция с amoCRM в V5

### DISCUSSION THREAD

**Discussion by gorbee** [totumonline/totum-ru-issues-and-discussions#600] [ОШБК!] [answered]
Привет!

В V5 при загрузке файла семпла из how-to [Подключаем Тотум к amoCRM](https://ru.docs.totum.online/totum-amo) возникает проблема 

```
<br />
<b>Notice</b>:  fwrite(): Send of 108 bytes failed with errno=32 Broken pipe in <b>/home/totum/totum-mit/totum/common/configs/ConfParent.php</b> on line <b>1115</b><br />
{"error":"GOMODULE error: Syntax error:``<br\/><br\/>"}
```

Инструкция несовместима с V5?

**Comment by gorbee** [comment]
Хотя таблица и данные из семпла загрузились в схему (заметил позже). Но, исходя из данной ошибки, можно ли пользоваться интеграцией? Не будет ли проблем в работе?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Это косметическая ошибка, ни на что не влияет. Позже поправим.
